src/options/options.go

```diff
@@ -125,3 +125,19 @@ func Initialize() error {
 	}
 	return nil
 }
+
+// CleanupDanglingImages - Cleans up all the dangling images
+func CleanupDanglingImages() {
```

Comment:
I am pretty sure this won't work. We use docker-machine to connect to the remote machines, and this will only clean up dangling images on the main VM (which is running deploybot) and not on the actual VMs (VM1 and VM2) which are running the services. See the deploy.sh code to see how we connect to remote VMs using docker-machine.